index.js

```diff
@@ -1,12 +1,31 @@
-const express = require('express');
+var express = require('express');
 var _ = require("underscore");
+var dataService = require('./dataService.js');
 
-const products = require('./src/data/products.json').products;
-const categories = require('./src/data/categories.json').categories;
+var app = express();
+var PORT = 8080;
 
-const app = express();
-const PORT = 8080;
+// Routes 
+app.get('/products/all', (req, res) => {
+  res.send(Array.from(dataService.getCombinedProductMap()));
```

Comment:
You should return a map in the form of `Object`, like: 

```json
{
  "P001": {
    "name": "...",
    "categoryId": "...",
    ...
  },
  "P002": {
    "name": "...",
    "categoryId": "...",
    ...
  }
}
```

and not an array like this.